front/pages/w/[wId]/assistant/labs/transcripts/index.tsx

```diff
@@ -130,7 +130,10 @@ export default function LabsTranscriptsIndex({
     return <Spinner />;
   }
 
-  const agents = agentConfigurations.filter((a) => a.status === "active");
+  // Only custom assistants - there's no point in passing a transcript to a default assistant without instructions.
+  const agents = agentConfigurations.filter(
+    (a) => a.status === "active" && a.id != -1
```

Comment:
Don't we have a function to filter them out?

Reply:
You likely can tweak that instead: agentsGetView: "list",

Reply:
Are we actually using agentsGetView? 

From the SWR `useAgentConfigurations` it looks like it, but looking at `front/pages/api/v1/w/[wId]/assistant/agent_configurations.ts` the GET request does not take into account any query param :/

```
const agentConfigurations = await getAgentConfigurations({
        auth,
        agentsGetView: "all",
        variant: "light",
      });
```

Reply:
oh it should cc @philipperolet who introduced it I believe?